scripts/include_folder.py

```diff
@@ -28,9 +28,20 @@ def remove_ddeb_from_changes_file(filename):
             if not l.strip().endswith('.ddeb'):
                 trimmed_version.append(l)
             else:
+                deb_path = os.path.dirname(filename)
+                # Checksum entries have three columns. Files entries have four.
+                # The last is always the filename.
+                ddeb_file = l.strip().split(' ')[-1]
+                basename = os.path.splitext(ddeb_file)[0]
+                deb_file = basename + '.deb'
+                # Rename the ddeb file if it hasn't been already.
+                if os.path.isfile(os.path.join(deb_path, ddeb_file)):
+                    print('Renaming `{}` to `{}`'.format(ddeb_file, deb_file))
```

Comment:
This would be a little clearer in the logs if it mentioned that it both renamed the file, and updated the changes file accordingly.

Reply:
The subsequent line in the log mentions mentions renaming the changes file entry. Excerpted from the example.

```
06:13:25 Renaming `ros-melodic-rospack-dbgsym_2.5.0-0bionic.20180522.131147_amd64.ddeb` to `ros-melodic-rospack-dbgsym_2.5.0-0bionic.20180522.131147_amd64.deb`
06:13:25 Renamed ddeb files to deb in changes file: /var/repos/ubuntu/building/queue/bionic/Mbin_uB64__rospack__ubuntu_bionic_amd64__binary__6/ros-melodic-rospack_2.5.0-0bionic.20180522.131147_amd64.changes to workaround https://github.com/ros-infrastructure/buildfarm_deployment/issues/186
```

Do you think that's sufficient or should we try and cram both onto one log line?

Reply:
It's probably adequate. It would only be useful to have each rename called out if we didn't trust that it would do the same thing as the actual rename. But I think that's unlikely something that we'll want to debug in the future.

Reply:
> It would only be useful to have each rename called out if we didn't trust that it would do the same thing as the actual rename.

I'm not sure I understand. Is the situation you have in mind that the renamed file on disk and the substituted filename in the changes file could be different?

Reply:
Yes, if you're just reading the logs. It will say I renamed files foo to bar, foo2 to baz, and foo3 to bar3. And then it will just say I changed some names in the changes file. If you're just reading the log you don't have an assurance that the changes to the changes file are equivalent to the files getting renamed which would be the symmetric verbose solution. The other cleaner way would be to state after each rename and changes update to state. For foo I renamed to bar and updated the .changes file accordingly so that it's know that the changes file was specifically updated for each item. It's implied from reading the current output but isn't positively confirmed per line. There is not a break in the control flow that could cause that to get out of sync, but to check that you need to look at the code. It's fine as is and we'll know what's going on I was just surprised to see it mutating the file w/o calling out the specific mutations but it did call out the specific file renames.

Reply:
I hear your concern and I think it's a valid one, though as you say with sufficient context it will be easy to investigate.

I'm weighing resolving this concern by updating the log line with the amount of work required to test this exhaustively and I think I'm going to opt to leaving it as is.